### PR TITLE
ILS: ScanRule Consistency

### PR DESCRIPTION
**Pull request by kingthorin**
- Scan Rule class names.
- Unit Test class names.
- CHANGELOG already contains a "maintenance changes" note.

Part of: zaproxy/zaproxy#6049

Signed-off-by: kingthorin <kingthorin@users.noreply.github.com>

### CODE REVIEW

File: addOns/imagelocationscanner/src/main/java/org/zaproxy/zap/extension/imagelocationscanner/ImageLocationScanRule.java [hunk 1]
```diff
@@ -48,9 +48,9 @@
  * @version 1.1
  * @see https://www.veggiespam.com/ils/
  */
-public class ImageLocationScanner extends PluginPassiveScanner {
+public class ImageLocationScanRule extends PluginPassiveScanner {
 	private PassiveScanThread parent = null;
-	private static final Logger logger = Logger.getLogger(ImageLocationScanner.class);
+	private static final Logger logger = Logger.getLogger(ImageLocationScanRule.class);
 	private static final String MESSAGE_PREFIX = "imagelocationscanner.";
```
Comment (thc202) [inline addOns/imagelocationscanner/src/main/java/org/zaproxy/zap/extension/imagelocationscanner/ImageLocationScanRule.java:54]: No changes to the key?

Comment (kingthorin) [reply addOns/imagelocationscanner/src/main/java/org/zaproxy/zap/extension/imagelocationscanner/ImageLocationScanRule.java:54]: Well I didn't think I should rename the whole add-on.

I guess the keys don't have to be named after the add-on though.

I'll update those that are used by the scan rule.

Comment (thc202) [reply addOns/imagelocationscanner/src/main/java/org/zaproxy/zap/extension/imagelocationscanner/ImageLocationScanRule.java:54]: Fine as is, just checking.

Comment (kingthorin) [reply addOns/imagelocationscanner/src/main/java/org/zaproxy/zap/extension/imagelocationscanner/ImageLocationScanRule.java:54]: Okay

**Comment by thc202** [comment]
Thanks!

**Pull request merged**